spring-cloud-dataflow-docs/src/main/asciidoc/getting-started-cloudfoundry.adoc

```diff
@@ -139,48 +139,6 @@ You can use the `--random-route` option to avoid this when you push the server a
 NOTE: If you need to configure multiple Maven repositories, a proxy, or authorization for a private repository, see link:http://docs.spring.io/spring-cloud-dataflow/docs/{scdf-core-version}/reference/htmlsingle/#getting-started-maven-configuration[Maven Configuration].
 
 
-[[getting-started-cloudfoundry-deploying-using-manifest]]
```

Comment:
Is there a reason to delete this?

Reply:
Hey Ilaya!    Its a duplicate section of what is found at line 244